crates/ide_completion/src/completions/postfix/format_like.rs

```diff
@@ -27,11 +27,11 @@ static KINDS: &[(&str, &str)] = &[
     ("panic", "panic!"),
     ("println", "println!"),
     ("eprintln", "eprintln!"),
-    ("logd", "log::debug!"),
-    ("logt", "log::trace!"),
-    ("logi", "log::info!"),
-    ("logw", "log::warn!"),
-    ("loge", "log::error!"),
+    ("logd", "tracing::debug!"),
```

Comment:
I'm not sure we want to change these.

Reply:
Right, I should not change these, sorry.

Reply:
Already reverted.